Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	(revision 69c6ab787bfd3fade763cb5db84287d28d77052c)
+++ b/index.html	(revision 5b19f99cc56e632ee1dd682fe4a2578230f788d0)
@@ -11,7 +11,7 @@
          href="https://goal-life.com/page/method/matrix-eisenhower">Want more details?
       </a>
   </div>
-  <div class="centered-main-page-element">
+  <div class="main-element">
       <p id="top-section-add-element"><label for="new-task">Add Item</label>
       <div class="task-row-wrapper">
       <input id="new-task"
Index: style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/style.css b/style.css
--- a/style.css	(revision 69c6ab787bfd3fade763cb5db84287d28d77052c)
+++ b/style.css	(revision 5b19f99cc56e632ee1dd682fe4a2578230f788d0)
@@ -25,7 +25,7 @@
 }
 }
 
-.centered-main-page-element {
+.main-element {
     display: block;
     width: 500px;
     margin: 0 auto 0;
