Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	(revision adb713659674c73e8a1ecbcf9a96160865c3dcfa)
+++ b/index.html	(revision a32804d339482ab83d545ea92323fdc23563a9e8)
@@ -7,7 +7,7 @@
 </head>
 <body>
   <div class="aaa"><img src="./eisenhower-matrix.jpg">
-      <a class="more_inf"
+      <a class="more-info"
          href="https://goal-life.com/page/method/matrix-eisenhower">Want more details?
       </a>
   </div>
Index: style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/style.css b/style.css
--- a/style.css	(revision adb713659674c73e8a1ecbcf9a96160865c3dcfa)
+++ b/style.css	(revision a32804d339482ab83d545ea92323fdc23563a9e8)
@@ -16,7 +16,7 @@
     width: 100%;
 }
 
-.aaa .more_inf {
+.aaa .more-info {
     font-family: fantasy, cursive;
 }
 
