Index: app.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app.js b/app.js
--- a/app.js	(revision a32804d339482ab83d545ea92323fdc23563a9e8)
+++ b/app.js	(revision 69c6ab787bfd3fade763cb5db84287d28d77052c)
@@ -10,7 +10,7 @@
 
 var taskInput=document.getElementById("new-task");//Add a new task.
 var addButton=document.getElementsByTagName("button")[0];//first button
-var incompleteTaskHolder=document.getElementById("incompleteTasks");//ul of #incompleteTasks
+var incompleteTaskHolder=document.getElementById("incomplete-tasks");//ul of #incompleteTasks
 var completedTasksHolder=document.getElementById("completed-tasks");//completed-tasks
 
 
@@ -85,7 +85,7 @@
     var editInput=listItem.querySelector("input[type=text]");
     var label=listItem.querySelector("label");
     var editBtn=listItem.querySelector(".edit");
-    var containsClass=listItem.classList.contains("editMode");
+    var containsClass=listItem.classList.contains("edit-mode");
     //If class of the parent is .editmode
     if(containsClass){
 
@@ -99,7 +99,7 @@
     }
 
     //toggle .editmode on the parent.
-    listItem.classList.toggle("editMode");
+    listItem.classList.toggle("edit-mode");
 };
 
 
Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	(revision a32804d339482ab83d545ea92323fdc23563a9e8)
+++ b/index.html	(revision 69c6ab787bfd3fade763cb5db84287d28d77052c)
@@ -6,13 +6,13 @@
     <link rel="stylesheet" href="style.css" media="screen" charset="utf-8">
 </head>
 <body>
-  <div class="aaa"><img src="./eisenhower-matrix.jpg">
+  <div class="main-matrix"><img src="./eisenhower-matrix.jpg">
       <a class="more-info"
          href="https://goal-life.com/page/method/matrix-eisenhower">Want more details?
       </a>
   </div>
   <div class="centered-main-page-element">
-      <p id="topSectionForAddingElementToList"><label for="new-task">Add Item</label>
+      <p id="top-section-add-element"><label for="new-task">Add Item</label>
       <div class="task-row-wrapper">
       <input id="new-task"
              class="task"
@@ -21,7 +21,7 @@
   </div>
       </p>
       <h3>Todo</h3>
-  <ul id="incompleteTasks">
+  <ul id="incomplete-tasks">
       <li><input type="checkbox">
           <label class="task">Pay Bills</label>
           <input type="text" class="task">
@@ -30,7 +30,7 @@
               <img src="./remove.svg">
           </button>
       </li>
-      <li class="editMode">
+      <li class="edit-mode">
           <input type="checkbox">
           <label class="task">Go Shopping</label>
           <input type="text"
Index: style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/style.css b/style.css
--- a/style.css	(revision a32804d339482ab83d545ea92323fdc23563a9e8)
+++ b/style.css	(revision 69c6ab787bfd3fade763cb5db84287d28d77052c)
@@ -5,23 +5,23 @@
     font-family: Lato, sans-serif;
 }
 
-.aaa {
+.main-matrix {
     width: 500px;
     margin: 0 auto;
     display: block;
     text-align: right;
 }
 
-.aaa img {
+.main-matrix img {
     width: 100%;
 }
 
-.aaa .more-info {
+.main-matrix .more-info {
     font-family: fantasy, cursive;
 }
 
 @media (max-width:768px) {
-.aaa { text-align: center;
+.main-matrix { text-align: center;
 }
 }
 
@@ -157,11 +157,11 @@
     display:none
 }
 
-ul li.editMode input[type=text] {
+ul li.edit-mode input[type=text] {
     display:inline-block;
     width:224px
 }
 
-ul li.editMode label {
+ul li.edit-mode label {
     display:none;
 }
\ No newline at end of file
